protocol/src/main/protobuf/Transactor.proto

```diff
@@ -14,6 +14,24 @@ message UpdateRequest {
     bytes chainCellCbor = 1;
 }
 
+message JournalStreamRequest {
+    MultihashReference lastJournalBlock = 1;
+}
+
+message UpdateChainResult {
+    MultihashReference canonicalRef = 1;
+    MultihashReference chainHeadRef = 2;
+    MultihashReference chainPreviousHeadRef = 3;
```

Comment:
How do you feel about `canonical`, `chain`, and `chainPrevious`?  It's closer to the ChainEntry naming but I think using `canonical` instead of `ref` is a bit clearer.   Maybe it's better to be consistent with ChainEntry though.